test/chompfile.toml

```diff
@@ -29,7 +29,7 @@ deps = [':test[0-9]*']
 [[task]]
 name = 'test1'
 display = 'none'
-run = '../target/debug/chomp output/test1.txt'
+run = 'chomp output/test1.txt'
```

Comment:
Again, this would use the global chomp instead of the local version for testing.